scalardb-test/src/main/java/kelpie/scalardb/Common.java

```diff
@@ -54,7 +55,12 @@ public static TwoPhaseCommitTransactionManager getTwoPhaseCommitTransactionManag
   }
 
   public static DatabaseConfig getDatabaseConfig(Config config) {
-    String configFile = config.getUserString("storage_config", "config_file", null);
+    String configFile;
+    try {
```

Comment:
It's a minor thing but is it slightly better to do as follows?
```
try {
  String configFile = config.getUserString("storage_config", "config_file");
  return new DatabaseConfig(new File(configFile));
} catch (Exception e) {
  LOGGER.warn("failed to load the specified config file: " + configFile, e);
}
```

Reply:
Thank you for taking a look!

After the change, we have two ways to specify database/storage info as follows:
```toml
[storage_config]
  config_file = "database.properties"
```
and
```toml
[storage_config]
  contact_points = "localhost"
  username = "cassandra"
  password = "cassandra"
  storage = "cassandra"
```

And if we apply the change you suggested and we use the latter way, the warning message is always shown.

Reply:
Ah, OK. Thanks!